### PR TITLE
added prettier test for Typescripts

### LINKED ISSUE #132: Add TypeScript code check for pre deploy stage
We should add pipeline stage for checking TypeScript code. Now we are using only `prettier` but maybe we can add more? 

Let's do only `prettier` for these issue anyway

### LINKED ISSUE #136: TypeScript code pipeline check stage
We should add pipeline stage to check TypeSript code quality. Make it allow to fail if it's possible to, you should not fix the code, only create the stage.

Can we have it in parallel with python code quality check? 

Also please use NPM docker image for it to speed it up if it's possible

    - npm run lint:no-fix
    - npm run prettier:no-fix

### PR DESCRIPTION
**Pull request by PyGeek33**
Closes #132
Closes #136

### CODE REVIEW

**Comment by PyGeek33** [comment]
everything is done

**Comment by Grommash9** [comment]
It's sad what we can't make it run in parallel (I tryed too)

Maybe we can speed it up?

We can make it to use the docker image what already have node installed, here is example:
Screenshot from 2024-01-10 08-16-41

So we will not need to install it every time, we are using `node:14-alpine` so it should be fine to use it in pipeline

Also we should break `.ts` code and check what pipeline will fail after it

We are don't really want making python devs to wait extra minute before their code check, let's try to make it as fast as we can

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@ python: 3.10.13
 if: branch = master AND type = push OR type = pull_request
 
 stages:
-  - name: pre-deploy
+  - name: pre-deploy typescript code checking
+  - name: pre-deploy code python checking
```
Comment (Grommash9) [inline .travis.yml:8]: I think we can use smaller name like

`linting python`
`linting ts`

but if you have any other shorted names ideas I will be happy with them

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +15,15 @@ before_script:
 
 jobs:
   include:
-    - stage: pre-deploy
+    - stage: pre-deploy typescript code checking
+      script:
+        - export PIPENV_VERBOSITY=-1
```
Comment (Grommash9) [inline .travis.yml:20]: Do we really need `PIPENV_VERBOSITY` variable here?

**Comment by Grommash9** [comment]
In pipeline we have a message about
`npm WARN exec The following package was not found and will be installed: prettier@3.1.1`

I think we should install our packages from repo and use it

Please check

`typescript-app/package.json`

**Comment by Grommash9** [comment]
Also would be great after you will done testing change places for ts and python, because the most of contributors are using python and let's no make them to wait them for ts check

**Review by Grommash9 — APPROVED** [review]

**Pull request merged**